posts/2022-04-01-cloud-native-liberty-buildpack.adoc

```diff
@@ -94,17 +94,49 @@ Set a default builder:
 [source]
 pack config default-builder gcr.io/paketo-buildpacks/builder:base
 
-Build the app:
+Build the application with only the Liberty features required to run the application and IBM Semeru OpenJ9:
 [source]
-pack build myapp --env BP_JAVA_APP_SERVER=liberty
+ pack build --env BP_JAVA_APP_SERVER=liberty --env BP_LIBERTY_PROFILE=kernel \  
+  --env BP_LIBERTY_FEATURES='jaxrs-2.1 jsonp-1.1 cdi-2.0 mpMetrics-3.0 mpConfig-2.0' \
+  --buildpack paketo-buildpacks/eclipse-openj9 --buildpack paketo-buildpacks/java myapp
 
 Your application is now transformed into an OCI image!
 
 == Now what? 
-With your OCI image, you can run your application locally with the `docker run` command:
+With your OCI image, you can run your application locally with the `docker run` command.  
 
+We need to provide a `server.xml` to the Liberty buildpack using a binding.  
+
+* Create a directory named `bindings`, for example
+* Create a file named server.xml in the `bindings` directory with the following content:
+[source]
+<server description="Intro to the Paketo Liberty buildpack">
+  <featureManager>
+      <!--feature>webProfile-8.0</feature -->
```

Comment:
it looks like the webProfile-8.0 feature is commented out- is that intentional?

Reply:
Ah...crap.  I'll remove it.